ios/FluentUI/ActivityIndicator/ActivityIndicatorTokenSet.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// Pre-defined sizes of the Activity Indicator.
+@objc public enum MSFActivityIndicatorSize: Int, CaseIterable {
+    case xSmall
+    case small
+    case medium
+    case large
+    case xLarge
+}
+
+/// Design token set for the `ActivityIndicator` control.
+open class ActivityIndicatorTokenSet: ControlTokenSet<ActivityIndicatorTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The default color of the Activity Indicator.
+        case defaultColor
+
+        /// The value for the side of the square frame of an Activity Indicator.
+        case side
+
+        /// The value for the thickness of the ActivityIndicator ring.
+        case thickness
+    }
+
+    /// MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
+    public internal(set) var size: MSFActivityIndicatorSize = .large
+
+    override func defaultValue(_ token: Tokens) -> ControlTokenValue {
+        switch token {
+        case .defaultColor:
+            return .dynamicColor { self.aliasTokens.foregroundColors[.neutral4] }
+
+        case .side:
+            return .float {
+                switch self.size {
+                case .xSmall:
+                    return 12
+                case .small:
+                    return 16
+                case .medium:
+                    return 24
+                case .large:
+                    return 32
+                case .xLarge:
+                    return 36
+                }
+            }
+
+        case .thickness:
```

Comment:
we don't have overall global size or thickness tokens? why is it a control token set?

Reply:
We do, but (A) they don't map 1:1 to these values, and (B) even if it did, we don't want to directly reference tokens in the control itself. All values should go through a `ControlTokenSet` for code cleanliness and to provide clean override hooks.